externs/shaka/player.js

```diff
@@ -1246,11 +1246,17 @@ shaka.extern.MediaSourceConfiguration;
 
 
 /**
- * @typedef {Object}
+ * @typedef {{
+ *   customPlayheadTracker: boolean
+ * }}
  *
  * @description
  *   Ads configuration.
  *
+ * @property {boolean} customPlayheadTracker
+ *   If this is <code>true</code>, we create a custom playhead tracker for
```

Comment:
It might be nice to have some details here on why/when you might want this.  In the PR description, too.

Reply:
Done!

Reply:
Ok, I have to admit I'm not sure what this has to do with having multiple video elements?
I tried the change out and it doesn't seem to result in the ad container having a different DOM structure than the regular method.

Reply:
The structure is the same, but the playhead tracker is used to get the position of the original video. If you can only have one video, if you delete it to leave the video to the ad, you stop having the position of the original video, so you can no longer track the position to launch the ads (vmap case)